codechat_config.json

```diff
@@ -0,0 +1,23 @@
+# ************************************************
+# |docname| - Configuration for a CodeChat project
+# ************************************************
+# This file defines the configuration for a CodeChat project. In addition to allowing standard JSON objects, it also allows Python-style comments and any valid Python literal.
+#
+# The file must contain a single dict, with the following keys:
+{
+    # ``source_path``: optional; defaults to ``.``. A path to the root of the source tree. Relative paths are rooted in the directory containing this file.
```

Comment:
DRY?  Duplicated information with what is in pavement.py/conf.py. We ought to look for ways to make these work together without duplication.

Reply:
Do you suggest loading this information from pavement? I'm not sure what you're thinking.